src/frontend/data/queries/updateOne.spec.tsx

```diff
@@ -0,0 +1,144 @@
+import fetchMock from 'fetch-mock';
+import React from 'react';
+
+import { updateOne } from './updateOne';
+
+let mockAppData = {};
+jest.mock('../appData', () => ({
+  get appData() {
+    return mockAppData;
+  },
+}));
+
+describe('queries/updateOne', () => {
+  beforeEach(() => (mockAppData = {}));
```

Comment:
```suggestion
  beforeEach(() => mockAppData = {});
```

Reply:
Prettier wants to keep those parenthesis.

Reply:
and this version works ? 

```
beforeEach(() => { mockAppData = {} });
```

Reply:
reformated like this, yes
```  
beforeEach(() => {
  mockAppData = {};
});
  ```